spec/operators/skipLast-spec.ts

```diff
@@ -132,11 +132,6 @@ describe('skipLast operator', () => {
     expectSubscriptions(e1.subscriptions).toBe(e1subs);
   });
 
-  it('should throw if total is less than zero', () => {
```

Comment:
I believe in this case I said that skipping "-1" would be the same as skipping 0. In that it would just mirror the source. Rendering the check useless.

I suppose that should be listed as a breaking change, though.